app/views/account_links/show.html.slim

```diff
@@ -1,39 +1,39 @@
 .header
   .std-heading
     i.fa-solid.fa-user
-    = " #{t('account.account_links.show_header')}: #{@account_link.name}"
+    = " #{AccountLink.model_name.human}: #{@account_link.name}"
 .show-table
   .row
-    .col-auto.row-label = t('account.account_links.name')
+    .col-auto.row-label = AccountLink.human_attribute_name('name')
     .col.row-value = @account_link.name
   .row
-    .col-auto.row-label = t('account.account_links.push_url')
+    .col-auto.row-label = AccountLink.human_attribute_name('push_url')
     .col.row-value = @account_link.push_url
   - if current_user == @account_link.user
     .row
-      .col-auto.row-label = t('account.account_links.check_uuid_url')
+      .col-auto.row-label = AccountLink.human_attribute_name('check_uuid_url')
       .col.row-value = @account_link.check_uuid_url
     .row
-      .col-auto.row-label = t('account.account_links.api_key')
+      .col-auto.row-label = AccountLink.human_attribute_name('api_key')
       .col.row-value = @account_link.api_key
     - if @account_link.shared_users.any?
       .row.vertical
-        .col.row-label = t('account.account_links.shared_users')
+        .col.row-label = t('account_links.show.shared_users')
         .row-value
           - @account_link.shared_users.each do |shared_user|
             .table-list
               = shared_user.email
               .btn-group.float-end
                 = button_to remove_shared_user_user_account_link_path, method: :post, params: {shared_user: shared_user.id}, data: { confirm: 'Are you sure?' }, class: 'btn btn-light btn-sm', remote: true, form_class: 'remove-account-link-form' do
                   i.fa-solid.fa-trash-can
-                  =< t('account.account_links.button.destroy')
+                  =< t('common.button.remove')
 
 .actions.btn-group
   - if can? :edit, @account_link
     = link_to edit_user_account_link_path(@user, @account_link), class: 'btn btn-important' do
       i.fa-solid.fa-pen-to-square
-      =< t('account.account_links.button.edit')
+      =< t('common.button.edit')
     = link_to user_account_link_path(@user, @account_link), class: 'btn btn-important', method: :delete, data: {confirm: 'Are you sure?'} do
```

Comment:
Here (and in line 27) is another confirmation that we can translate (and should, ideally, search for those globally).

Reply:
I did a quick search and replaced all remaining occurrences of that (there were multiple).